lib/util/log.sh

```diff
@@ -6,6 +6,9 @@ declare -Ag __oo__logScopeOutputs
 declare -Ag __oo__logDisabledFilter
 declare -Ag __oo__loggers
 
+# Controls verbosity of the script output and logging.
+VERBOSE="${VERBOSE:-5}"
```

Comment:
I'd use something like `__oo__verbosity` for the variable name, `VERBOSE` can conflict with a user-space variable of the same name.

Ideally, we'd set this variable with some configuration function, but I can accept this as a temporary workaround.

Reply:
the variable `VERBOSE` is for user to set.
Just like the example, before or after users `import util/log`, they can set/change the `VERBOSE` level by themselves to determine which log will be displayed
```
VERBOSE=2

Log::AddOutput test/verbose DEBUG

Log "this log will be displayed"
V=2 Log "this log will be displayed"

VERBOSE=1
V=2 Log "this log will not be displayed"
```